tests/c_languages/CPPLintBearTest.py

```diff
@@ -13,16 +17,42 @@
                                     tempfile_kwargs={"suffix": ".cpp"})
 
 CPPLintBearIgnoreConfigTest = verify_local_bear(
-    CPPLintBear,
-    valid_files=(test_file,),
-    invalid_files=(),
-    settings={'cpplint_ignore': 'legal'},
-    tempfile_kwargs={"suffix": ".cpp"})
+        CPPLintBear,
+        valid_files=(test_file,),
+        invalid_files=(),
+        settings={'cpplint_ignore': 'legal'},
+        tempfile_kwargs={"suffix": ".cpp"})
 
 CPPLintBearLineLengthConfigTest = verify_local_bear(
-    CPPLintBear,
-    valid_files=(),
-    invalid_files=(test_file,),
-    settings={'cpplint_ignore': 'legal',
-              'max_line_length': '13'},
-    tempfile_kwargs={"suffix": ".cpp"})
+        CPPLintBear,
+        valid_files=(),
+        invalid_files=(test_file,),
+        settings={'cpplint_ignore': 'legal',
+                  'max_line_length': '13'},
+        tempfile_kwargs={"suffix": ".cpp"})
+
+
+@generate_skip_decorator(CPPLintBear)
+class CPPLintBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section("test section")
+        # uut = unit under test
+        self.uut = CPPLintBear(self.section, Queue())
+
+    def test_config_failure_use_spaces(self):
+        self.section["use_spaces"] = "False"
+        self.section.append(Setting('use_spaces', "False"))
```

Comment:
If I'm not mistaken, you just neet the `self.section.append(Setting('use_spaces', "False"))`, not the `self.section["use_spaces"] = "False"`.
This goes for the other test cases as well